### ISSUE TITLE
Juxtapose doesn't showing on Chrome Browser

### ISSUE THREAD

**Issue by heavenly-roads** [NUKnightLab/juxtapose#149]
I've noticed iframe images are not showing on Google Chrome browser (chrome version 73.0.3683.103). Instead, it shows an empty space.

**Comment by JoeGermuska** [comment]
please provide a test case (URL) that demonstrates the problem. There's a high likelihood that the problem is Chrome refusing to load content from `http` URLs when the containing page is loaded from an `https` URL.  At least, when I go to juxtapose.knightlab.com on the same version of Chrome, I don't experience the problem you describe.

**Comment by heavenly-roads** [comment]
@JoeGermuska Right now I can't provide a URL, due is being tested internally on my website and that part it's not public opened. But you're right - my website is https and maybe Chrome refuses to load it. Thank you for your comment.

**Comment by JulianGro** [comment]
Same problem here on Vivaldi. Someone sent me this: https://cdn.knightlab.com/libs/juxtapose/latest/embed/index.html?uid=c4df8da2-da22-11eb-b7bf-95443c729a29
I assume it is just imgur blocking the embed somehow, but I can load the images directly so I don't see what's going on.

**Comment by JoeGermuska** [comment]
It is against Imgur's terms of service to "hotlink" images, which is what is required to use images in JuxtaposeJS.

It may appear to work while creating, but that's because your browser has the images cached. When others, or you after your cache clears, load the Juxtapose, the browser network inspector will show `403 Forbidden` responses from Imgur for those images:

<img width="511" alt="image" src="https://user-images.githubusercontent.com/46313/124138816-a5dc9b00-da4c-11eb-8a62-a0b937386455.png">

Put simply: you can't use Imgur for JuxtaposeJS.  For creators who are not able to simply put the images on a website they control, Flickr and Dropbox are supported by Juxtapose. There are also general image hosting services such as https://postimages.org/ which can be used.
